### PR TITLE
dhcp: Validate dhcp option 60 is being logged

### PR DESCRIPTION
**Pull request by yatink**
## Feature 4587
..continuation of #1355 

This PR adds a test to validate that dhcp option 60 is being logged in dhcp extended logging mode.

Associated suricata PR : https://github.com/OISF/suricata/pull/9397
Redmine ticket: https://redmine.openinfosecfoundation.org/issues/4587

### CODE REVIEW

File: tests/dhcp-eve-extended-option-60/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Test Description
+
+Valide that dhcp option 60 (vendor class identifier) is being logged by suricata
```
Comment (inashivb) [inline tests/dhcp-eve-extended-option-60/README.md:3]: super nit: Typo `Valide`

**Review by inashivb — CHANGES_REQUESTED** [review]
LGTM. Just a minor typo fix needs TBD. :)

**Comment by yatink** [comment]
Closing in favour of a new one in that case.

**Comment by yatink** [comment]
Continued in #1361

**Pull request closed without merging**